tests/test_websockets.py

```diff
@@ -281,5 +281,5 @@ def test_websocket_scope_interface():
         send=mock_send,
     )
     assert websocket["type"] == "websocket"
-    assert dict(websocket) == {"type": "websocket", "path": "/abc/", "headers": []}
-    assert len(websocket) == 3
+    assert dict(websocket) == {"type": "websocket", "path": "/abc/", "headers": [], 'state': {}}
```

Comment:
This won't have changed once we *only* set `state` to `{}` if `.state` is accessed.